examples/MvcWeb/TheraLang/Repository/ProjectRepository.cs

```diff
@@ -7,7 +7,7 @@ namespace MvcWeb.TheraLang.Repository
 {
     public class ProjectRepository : IProjectRepository
     {
-        IUnitOfWork uow;
+        IUnitOfWork uow = new MvcWeb.UnitOfWork.UnitOfWork();
```

Comment:
😤😵
Damn, this ProjectRepository is not used. Could you please remove those files at all?
IProjectRepository.cs, ProjectRepository.cs